optuna/multi_objective/_hypervolume/wfg.py

```diff
@@ -23,6 +24,14 @@ def _compute(self, solution_set: np.ndarray, reference_point: np.ndarray) -> flo
     def _compute_rec(self, solution_set: np.ndarray) -> float:
         assert self._reference_point is not None
         n_points = solution_set.shape[0]
+        _shape = self._reference_point.shape
```

Comment:
I'm not familiar with the leading underscore, is it required here?

Reply:
Thank you for pointing out, it was added without any considering.
6baf049 removed leading underscores for ordinal variables.